### PR TITLE
Add hex encoded rgb to colorpickers tooltips

### PR DESCRIPTION
**Pull request by jenshannoschwalm**
Inspired by #4611 this does not implement the whole "wish list" but it adds
the rgb hex value to the tooltip.

### CODE REVIEW

File: src/libs/colorpicker.c [hunk 1]
```diff
@@ -175,7 +175,10 @@ static void _update_picker_output(dt_lib_module_t *self)
     switch(input_color)
     {
       case 0: // rgb
-        snprintf(tooltip, sizeof(tooltip), "%3d   %3d   %3d",
+        snprintf(tooltip, sizeof(tooltip), "%3d   %3d   %3d   %02X%02X%02X",
```
Comment (TurboGit) [inline src/libs/colorpicker.c:178]: Can we add some kind of separator here ? Maybe "%3d   %3d   %3d  - %02X%02X%02X" or using | ?

Or maybe "hex: " to clearly indicate that the values are hex encoded ?

File: src/libs/colorpicker.c [hunk 2]
```diff
@@ -290,7 +293,10 @@ static void _update_samples_output(dt_lib_module_t *self)
     {
       case 0:
         // RGB
-        snprintf(tooltip, sizeof(tooltip), "%3d   %3d   %3d",
+        snprintf(tooltip, sizeof(tooltip), "%3d   %3d   %3d   %02X%02X%02X",
```
Comment (TurboGit) [inline src/libs/colorpicker.c:296]: Likewise.

File: src/libs/colorpicker.c (hunk 1 above)
Comment (jenshannoschwalm) [reply src/libs/colorpicker.c]: Yeah that is better. Changed to `0   128   255  (0x0080FF)`

File: src/libs/colorpicker.c (hunk 2 above)
Comment (jenshannoschwalm) [reply src/libs/colorpicker.c]: Sure

**Comment by TurboGit** [comment]
Works for me now. Thanks.

**Pull request merged**